Core1Platform/BasicPhilosophy/04_ModuleOverview.md

```diff
@@ -0,0 +1,115 @@
+With the *Omni-Channel* module, you can connect your web store and all relevant marketplaces directly to the platform and manage them centrally from it. You can integrate various store systems such as Shopware 6&trade;, Shopify&trade; or Amazon&trade;.
+It provides orders for the *Order Management System* (OMS) and authorizations for the *Payments* module.
+
+**POS**  
+With the cloud-based POS system of the *Actindo Core1 Platform*, you not only control your online channels via Actindo, but can also easily integrate your offline stores. The *Venduo POS* module enables digital and centralized management of your physical stores directly through the platform, dissolving obstructive boundaries between digital and classic commerce.
+You can integrate POS-Systems with multi location and multi cash register handling. It provides payment data to be posted in the *Accounting* module. 
+
+
+
+## Order Management
+
+
+**Warehousing**   
+The *Warehouse Management* is the central point on *Actindo's Core1 platform* for organizing and monitoring warehouse processes. It stores basic data needed for order management as well as for warehouse handling. You can use it to manage stock levels per material. It allows several kinds of bookings from the *Order Management System* (OMS) such as inventory, reservation, pickings, sells, and much more. 
+The module provides a complete inventory overview of all your products and warehouses, as well as inventory tracking in real time. You can plan warehouse locations according to different metrics such as building shapes and sizes, shelves or compartments, as well as manage individual storage bins. The module receives the material master data and provides stock levels for them. It receives stock updates from the *Fulfillment* module.  
+
+**Customers and Suppliers**    
+After orders are imported from any connected stores and marketplaces, customer master data is automatically created, or existing master data records are recognized and updated as needed. It provides customer data for the POS systems and receives updates from it.
+
+
+**Accounting**    
+The *Actindo Core1 Platform* is the only digital operations platform worldwide with a complete accounting module integrated. This means that all postings for all orders of the connected channels and accounts are automatically generated and optionally transferred live to a group accounting system.   
+The module stores any information on financial transactions. POS systems post payments directly to the *Accounting* module. It receives transactions from the *Payment Processing* module. 
+The *Accounting* module gets postings from the *Order Management System* and sends back updates on the payment status to ensure that no orders are delivered that are not paid.
```

Comment:
Payment processing, Order management, Customer and suppliers, DataHub importer / exporter... 
Groß-/Kleinschreibung beachten und vereinheitlichen

Reply:
wir müssen mal über Eigennamen reden